tt_metal/hw/ckernels/wormhole_b0/common/inc/ckernel_defs.h

```diff
@@ -130,27 +144,25 @@ constexpr static std::int32_t MUL_DEST_TILE_SIZE_AND_INDEX(uint format, uint ind
     };
 }
 
-// 16B words, L1/dram headerless!
 constexpr static std::int32_t GET_L1_TILE_SIZE(uint format) {
-    switch (format&0x1F) {
-        case ((uint8_t)DataFormat::Float32): return ((4096>>4));
+    switch (format&0xF) {
+        case ((uint8_t)DataFormat::Float32): return ((4096>>4)+(32>>4));
         case ((uint8_t)DataFormat::Float16):
-        case ((uint8_t)DataFormat::Float16_b): return ((2048>>4));
+        case ((uint8_t)DataFormat::Float16_b): return ((2048>>4)+(32>>4));
         case ((uint8_t)DataFormat::Bfp8):
-        case ((uint8_t)DataFormat::Bfp8_b): return ((1024>>4)+(64>>4));
+        case ((uint8_t)DataFormat::Bfp8_b): return ((1024>>4)+(64>>4)+(32>>4));
         case ((uint8_t)DataFormat::Bfp4):
-        case ((uint8_t)DataFormat::Bfp4_b): return ((512>>4)+(64>>4));
+        case ((uint8_t)DataFormat::Bfp4_b): return ((512>>4)+(64>>4)+(32>>4));
         case ((uint8_t)DataFormat::Bfp2):
-        case ((uint8_t)DataFormat::Bfp2_b): return ((256>>4)+(64>>4));
+        case ((uint8_t)DataFormat::Bfp2_b): return ((256>>4)+(64>>4)+(32>>4));
         case ((uint8_t)DataFormat::Int8):
-        case ((uint8_t)DataFormat::Lf8): return ((1024>>4));
-        default: return ((1024>>4)+(64>>4));
+        case ((uint8_t)DataFormat::Lf8): return ((1024>>4)+(32>>4));
+        default: return ((1024>>4)+(64>>4)+(32>>4));
     };
 }
 
-
 constexpr static std::int32_t GET_DEST_TILE_BYTE_SIZE(uint format) {
-    switch (format&0x1F) {
+    switch (format&0xF) {
```

Comment:
same with this.  it may be future work, but it would really be great to keep the magic to a minimum so others can read the code, something like "insn_format_mask"...